plugin/src/main/java/org/jenkinsci/plugins/structs/describable/UninstantiatedDescribable.java

```diff
@@ -0,0 +1,218 @@
+package org.jenkinsci.plugins.structs.describable;
+
+import hudson.model.Describable;
+import org.jenkinsci.Symbol;
+
+import javax.annotation.Nullable;
+import java.io.Serializable;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.TreeMap;
+
+/**
+ * A reflection-like view of a particular {@link Describable} instance.
+ *
+ * @author Kohsuke Kawaguchi
+ */
+public class UninstantiatedDescribable implements Serializable {
+    private String symbol;
+    private String klass;
+    private final Map<String,?> arguments;
+    private transient DescribableModel model;
```

Comment:
For instantiation this field is not used, so your code fragment above will work as expected. See comment in 8b12c663d2db24402b392f4abc4cdd43d4d87830. I'll leave this as a comment in code.